github/prci.py

```diff
@@ -4,140 +4,37 @@
 import logging
 import logging.config
 import os
-import random
-import raven
 import signal
-import subprocess
 import sys
-import time
-import yaml
+from functools import partial
+from time import sleep
 
 import github3
-import redis
-
-sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..'))
-from prci_github import TaskQueue, AbstractJob, JobResult
-from prci_github.adapter import GitHubAdapter
-
-
-SENTRY_URL = 'https://d24d8d622cbb4e2ea447c9a64f19b81a:4db0ce47706f435bb3f8a02a0a1f2e22@sentry.io/193222'
-ERROR_BACKOFF_TIME = 600
-REBOOT_DELAY = 3600 * 3
-REBOOT_TIME_FILE = '/root/next_reboot'
-logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
-
-
-class ExitHandler(object):
-    done = False
-    aborted = False
-    task = None
-
-    def finish(self, signum, frame):
-        if self.done:
-            return self.abort(signum, frame)
-
-        logger.info("Waiting for current task to finish. This may take long.")
-        self.done = True
-
-    def abort(self, signum, frame):
-        if self.aborted:
-            return self.quit()
-
-        if self.task:
-            self.task.abort()
-            logger.info("Waiting for aborted task to clean up. This may take "
-                        "few minutes.")
-
-        self.done = True
-        self.aborted = True
+import yaml
 
-    def quit(self):
-        logger.info("Quiting just now! No results will be reported.")
-        sys.exit()
+import internals.entities as entities
+from internals.gql import h, q
```

Comment:
As I said about the file names, I would prefer to have variables with a real name instead of just a char.

Reply:
`util` and `queries` maybe?

Reply:
`util` and `queries` it is